packages/storybook/cxl-ui/cxl-playbook-accordion/theme=cxl-playbook.faq.stories.js

```diff
@@ -0,0 +1,57 @@
+import { html } from 'lit-html';
+import '@conversionxl/cxl-ui/src/components/cxl-app-layout.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import faqData from '../cxl-vaadin-accordion/theme=cxl-faq.data.json';
+
+export default {
+  title: 'CXL UI/cxl-playbook-accordion',
+};
+
+const Template = () => html`
+  <style>
+    .plural .entry-title {
+      margin: 0;
+    }
+    cxl-playbook-accordion {
+      padding: 10px;
+    }
+  </style>
+
+  <cxl-app-layout theme="2c-l" id="container">
+    <cxl-playbook-accordion
+      id="cxl-vaadin-accordion-26107"
+      theme="reverse"
+      class="cxl-playbook-accordion-class"
+    >
+      ${faqData.map(
+        (el) => html`
+          <vaadin-accordion-panel
+            id="${el.cxl_hybrid_attr_post['@attributes'].id}"
+            class="${el.cxl_hybrid_attr_post['@attributes'].class}"
+            theme="reverse"
+          >
+            <div slot="summary">
+              <div class="left">
+                <vaadin-checkbox value="Option" theme="custom"></vaadin-checkbox>
+              </div>
+              <div class="right">
+                <div class="summary-top">${unsafeHTML(el.title.rendered)}</div>
+              </div>
```

Comment:
If you're already using `vaadin-vertical-layout`, is it possible to replace custom divs with `vaadin-horizontal-layout` here?

Reply:
refactored to use it in the last commit: https://github.com/conversionxl/aybolit/pull/87/commits/878a7c86be34f65fb67699d731d95fcc843c3234